### PR TITLE
adding travel grant information

### PR DESCRIPTION
**Pull request by jrgochan**
## Checklist:

- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [x] I have updated the README.md if necessary

cc **ADD GITHUB HANDLES HERE**
@jsubida

### CODE REVIEW

**Review by jrgochan — COMMENTED** [review]
I feel like I am making many mistakes.

File: pages/attend/travel.md [hunk 1]
```diff
@@ -37,6 +37,17 @@ US-RSE'24.
 
 - Hotel Andaluz Albuquerque, Curio Collection - 330 Tijeras Ave NW, Albuquerque, NM 87102 (0.1 miles away)
 
+## Travel Grants
+
+The US-RSE ’24 conference will again be offering travel grants.
```
Comment (mrmundt) [inline pages/attend/travel.md:42]:
```suggestion
The US-RSE'24 conference will again be offering travel grants via an application process.
```

File: pages/attend/travel.md [hunk 2]
```diff
@@ -37,6 +37,17 @@ US-RSE'24.
...
+
+- Travel grants can be used for transportation, registration, and/or accommodations.
+- Meals will be provided at the conference on conference days.
+- Priority will be given...
+  - to those having an accepted submission at the US-RSE’24 conference.
+  - to enhance the diversity of participants.
```
Comment (mrmundt) [inline pages/attend/travel.md:48]:
```suggestion
- Priority will be given to...
  - those with an accepted submission at the US-RSE’24 conference.
  - enhance the diversity of participants.
```

**Review by mrmundt — CHANGES_REQUESTED** [review]

**Review by jrgochan — COMMENTED** [review]
Looks good to me.

**Comment by KristinaRiemer** [comment]
Same! Any last improvements @mrmundt?

File: pages/attend/travel.md [hunk 3]
```diff
@@ -37,6 +37,17 @@ US-RSE'24.
...
 - Hotel Andaluz Albuquerque, Curio Collection - 330 Tijeras Ave NW, Albuquerque, NM 87102 (0.1 miles away)
 
+## Travel Grants
+
+The US-RSE ’24 conference will again be offering travel grants via an application process.
+
+- Travel grants can be used for transportation, registration, and/or accommodations.
+- Meals will be provided at the conference on conference days.
+- Priority will be given to...
+  - those having an accepted submission at the US-RSE’24 conference.
```
Comment (lmilechin) [inline pages/attend/travel.md:47]: I have a nitpick: do we want "US-RSE’24" or "US-RSE ’24" (with/without the space)? Right now we have one of each.

Comment (jsubida) [reply pages/attend/travel.md:47]: I vote no space. Other pages (as well as the header) have "US-RSE’24".

Comment (danielskatz) [reply pages/attend/travel.md:47]: no space is what we've used generally

**Comment by jsubida** [comment]
> I feel like I am making many mistakes.

["You really can't make a mistake here. Pull requests may or may not reveal happy accidents." - maybe Bob Ross if he did code](https://www.youtube.com/watch?v=Ht9LMYivBnM&t=19s)

File: pages/attend/travel.md [hunk 4]
```diff
@@ -37,6 +37,17 @@ US-RSE'24.
 
 - Hotel Andaluz Albuquerque, Curio Collection - 330 Tijeras Ave NW, Albuquerque, NM 87102 (0.1 miles away)
 
+## Travel Grants
+
+The US-RSE ’24 conference will again be offering travel grants via an application process.
```
Comment (KristinaRiemer) [inline pages/attend/travel.md:42]:
```suggestion
The US-RSE’24 conference will again be offering travel grants via an application process.
```

File: pages/attend/travel.md (hunk 3 above)
Comment (KristinaRiemer) [reply pages/attend/travel.md:47]: Based on consensus, I removed the space from the one "US-RSE" that had it.

File: .vscode/extensions.json
Comment (mrmundt) [inline .vscode/extensions.json:1]: I think maybe this file isn't supposed to be in this PR?

**Review by mrmundt — APPROVED** [review]

**Pull request merged**